### ISSUE TITLE
Delete Events for Point Triggers are weird

### ISSUE THREAD

**Issue by Bastian2718** [mautic/mautic#14412]
### Mautic Series

5.2.x series

### Mautic installed version

5.2.0

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.2.22

### What browsers are you seeing the problem on?

Chrome

### What happened?

Delete and Edit Button disappears sometimes. Or deletes the wrong event.
Also with 5.1.1.

https://github.com/user-attachments/assets/abf49212-4463-4870-a122-76bc68369b5e

### How can we reproduce this issue?

Step 1: Create a new Point Trigger
Step 2: Add 2 or more events
Step 3: Hover over an event
Step 4: Move the Cursor towards the delete Symbol that appeared in Step 3 with different angles (diagonal it disappears, straight up it does not disappear)
Step 5: Try to delete the 2nd one, use different angles too (diagonal it changes the event that should delete by clicking the button, Straight up didn't change the event even with pausing on the other Event)

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
Hi @Bastian2718 thanks for the bug report, but please make sure you search before raising an issue. This one is a duplicate of https://github.com/mautic/mautic/issues/13817 and there's a PR to fix here: https://github.com/mautic/mautic/pull/14448 - please help by testing, check out https://mau.tc/tester for how to do that! Thanks!

**Comment by RCheesley** [comment]
Re-opening this, @Bastian2718 please could you add your video here? @andersonjeccel I think this is a UI/UX issue.

**Comment by andersonjeccel** [comment]
Definitely a UI bug

**Comment by escopecz** [comment]
@andersonjeccel are you working on this or can I unassign you so someone else could take it?

**Comment by rishithreddy89** [comment]
@RCheesley  @escopecz  @andersonjeccel  if no one is working on this issue i want to work on this

**Comment by RCheesley** [comment]
@rishithreddy89 please go ahead!

**Comment by rishithreddy89** [comment]
I will raise a pr tonight.
